examples/asa/scripts/3-contract-owned-asa.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Description:
+ * This file demonstrates how to create ASA owned by
+ * smart contract account(stateless).
+ */
+const { executeTransaction } = require('@algo-builder/algob');
+const { mkParam } = require('./transfer/common');
+const { types } = require('@algo-builder/runtime');
+
+async function run (runtimeEnv, deployer) {
+  const masterAccount = deployer.accountsByName.get('master-account');
+  const alice = deployer.accountsByName.get('alice');
+
+  await executeTransaction(deployer, mkParam(masterAccount, alice.addr, 200e6, { note: 'funding account' }));
+
+  // Create Application
+  // Note: An Account can have maximum of 10 Applications.
+  const sscInfo = await deployer.deploySSC(
+    '5-contract-owned-asa-approval.py', // approval program
+    '5-clear.py', // clear program
+    {
+      sender: alice,
+      localInts: 1,
+      localBytes: 1,
+      globalInts: 1,
+      globalBytes: 1
+    }, {});
+
+  console.log(sscInfo);
+
+  // Get Statless Account Address
+  const statelessAccount = await deployer.loadLogic('5-contract-owned-asa.py', [], { APP_ID: sscInfo.appID });
+  console.log('stateless Account Address:', statelessAccount.address());
+
+  await executeTransaction(deployer, mkParam(masterAccount, statelessAccount.address(), 200e6, { note: 'funding account' }));
+
+  const txGroup = [
+    // Stateful call
+    {
+      type: types.TransactionType.CallNoOpSSC,
+      sign: types.SignType.SecretKey,
+      fromAccount: alice,
+      appId: sscInfo.appID,
+      payFlags: {}
+    },
+    // Asset creation
+    {
+      type: types.TransactionType.DeployASA,
+      sign: types.SignType.LogicSignature,
+      fromAccount: { addr: statelessAccount.address() },
```

Comment:
this should be `fromAccountAddr: statelessAccount.address()` now. Could you please pull from latest master & update?